packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
 # Sentry Config File
```

Comment:
Weird behavior with passkeys: 

1 - When I click SignIn the passkeys selector it's prompted
2 - I select a passkey
3 - A different email will appear on the input
4 - Click login
5 - User will be logged in with the email selected on passkey

https://drive.google.com/file/d/1HY-5EK0bk1SciU72sHFAf1pk7mSmYoJI/view?usp=sharing

The passkey I choose here it's from Apple Profile, not Chrome. Don't know if it makes difference

Reply:
@esaminu